os/Makefile.unix

```diff
@@ -295,6 +295,7 @@ endif
 # tools/mkversion tool is built and used to create include/tinyara/version.h
 
 force_build:
+	$(Q) $(TOPDIR)/tools/mk_heap_regioninfo.sh
```

Comment:
Please make a new tab to execute this like version and config.

Reply:
done.